### ISSUE TITLE
HELP! low accuracy on once dataset using pointrcnn.

### ISSUE THREAD

**Issue by DurbinLiu** [open-mmlab/OpenPCDet#1365]
I trained pointrcnn on once dataset for 80 epochs with the default config.
But got very poor accuracy as follows:
![image](https://github.com/open-mmlab/OpenPCDet/assets/66539992/a2018246-681d-4671-b019-09d7f35e209e)
I wonder where's the problem, did I set the config wrong?
could u please give me some advice on how to improve the performance?
Many thanks!

**Comment by PointsCoder** [comment]
The scores are as expected and similar to what was reported in the [paper](https://arxiv.org/pdf/2106.11037.pdf).

To further improve, you may try changing to a smaller batch size or adjusting the ball query radius, which might help a little.

**Comment by DurbinLiu** [comment]
> 

Hi, thanks for your reply, why smaller batch size would help? do you mean set a bigger batch size?

**Comment by PointsCoder** [comment]
I didn't try but I heard using batch size 1 can boost the AP by 2%
https://github.com/PointsCoder/ONCE_Benchmark/issues/31#issuecomment-1311804549

**Comment by PointsCoder** [comment]
I guess it is because the training set is small so different batch sizes bring some randomness on the performances.
